WorkflowTesting.podspec

```diff
@@ -19,6 +19,7 @@ Pod::Spec.new do |s|
     s.source_files = 'WorkflowTesting/Sources/**/*.swift'
 
     s.dependency 'Workflow', "#{s.version}"
+    s.dependency 'CustomDump', '~> 0.6.1'
```

Comment:
my guess is this is not going to work as a change to the public repo because this has been forked and hosted internally, but AFAIK does not exist in the public CocoaPods spec repo

Reply:
https://github.com/square/workflow-swift/actions/runs/5006839279/jobs/8973201477?pr=214#step:5:8

Reply:
I came across this and it seems public https://github.com/Golface/CocoaPods-Composable-Architecture

Reply:
Its public in terms of a public GH repo is having a podspec. Its not public as in "published in the CocoaPods open source registry" (or trunk). 

You can always double check if a pod is actually published in the CocoaPods registry by visiting cocoapods.org and searching for a pods name. If it appears then its published.

You will not be able to do:
```ruby
pod 'CustomDump', :git => `https://github.com/Golface/CocoaPods-Composable-Architecture.git`
```

as CocoaPods does not allow that.

Reply:
@DJBen this is a pain...we'd have to ask for someone to publish this in the open source repo or we cannot depend on it. An alternative is to duplicate all of it inside Workflow which is meh. Trying to think of alternatives..........

Reply:
If I publish it to the trunk, will it solve the issue? Because SQ internally is using a fork, will that cause a version conflict?

Reply:
If you do it would solve this issue here I believe yes. 

For SQ if it uses an internal fork it would have to ensure the version requirements specified here match and if they do then I think it will take the fork

Reply:
I tried to untangle the public dependency and here's what I find:
- `CustomDump` depends on `XCTestDynamicOverlay`, but it needs > 0.4. Unfortunately, an old 0.1.1 is occupied by another person github.com/Arafa.

Is it okay that I alias the pods with `PFC` (pointfreeco), so they become `PFCXCTestDynamicOverlay` and `PFCCustomDump`, and so I can upload our versions to the public trunk?

Reply:
@gilgtm I managed to contact Rafa and persuade him to delete the `XCTestDynamicOverlay` repo so I can push a new version. I managed to push `CustomDump 0.11.0` to cocoapods.